pkg/operator/workloadcontroller/workload_controller.go

```diff
@@ -1,295 +0,0 @@
-	kubeInformersForOpenShiftAPIServerNamespace.Core().V1().Services().Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForOpenShiftAPIServerNamespace.Apps().V1().Deployments().Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForKubeSystemNamespace.Core().V1().ConfigMaps().Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForOpenShiftConfigNamespace.Core().V1().ConfigMaps().Informer().AddEventHandler(c.eventHandler())
-	configInformers.Config().V1().Images().Informer().AddEventHandler(c.eventHandler())
-	apiregistrationInformers.Apiregistration().V1().APIServices().Informer().AddEventHandler(c.eventHandler())
-
-	// we only watch some namespaces
-	kubeInformersForOpenShiftAPIServerNamespace.Core().V1().Namespaces().Informer().AddEventHandler(c.namespaceEventHandler())
-
-	return c
-}
-
-func (c OpenShiftAPIServerOperator) sync() error {
-	operatorConfig, err := c.operatorConfigClient.OpenShiftAPIServers().Get("cluster", metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-
-	switch operatorConfig.Spec.ManagementState {
-	case operatorsv1.Managed:
-	case operatorsv1.Unmanaged:
-		return nil
-	case operatorsv1.Removed:
-		// TODO probably need to watch until the NS is really gone
-		if err := c.kubeClient.CoreV1().Namespaces().Delete(operatorclient.TargetNamespace, nil); err != nil && !apierrors.IsNotFound(err) {
-			return err
-		}
-		return nil
-	default:
-		c.eventRecorder.Warningf("ManagementStateUnknown", "Unrecognized operator management state %q", operatorConfig.Spec.ManagementState)
-		return nil
-	}
-
-	kubeAPIServerOperator, err := c.openshiftConfigClient.ClusterOperators().Get("kube-apiserver", metav1.GetOptions{})
-	if apierrors.IsNotFound(err) {
-		kubeAPIServerOperator, err = c.openshiftConfigClient.ClusterOperators().Get("openshift-kube-apiserver-operator", metav1.GetOptions{})
-	}
-	if apierrors.IsNotFound(err) {
-		message := "clusteroperator/kube-apiserver not found"
-		c.eventRecorder.Warning("PrereqNotReady", message)
-		return fmt.Errorf(message)
-	}
-	if err != nil {
-		return err
-	}
-	if !clusteroperatorv1helpers.IsStatusConditionTrue(kubeAPIServerOperator.Status.Conditions, "Available") {
-		message := fmt.Sprintf("clusteroperator/%s is not Available", kubeAPIServerOperator.Name)
-		c.eventRecorder.Warning("PrereqNotReady", message)
-		return fmt.Errorf(message)
-	}
-
-	// block until config is obvserved
-	if len(operatorConfig.Spec.ObservedConfig.Raw) == 0 {
-		klog.Info("Waiting for observed configuration to be available")
-		return nil
-	}
-
-	// block until extension-apiserver-authentication configmap is fully populated to avoid
-	// that openshift-apiserver starts up with request header setting (which are not dynamically reloaded).
-	// in the future we need to change upstream code to be more dynamic
-	// see https://bugzilla.redhat.com/show_bug.cgi?id=1795163#c19 for more details.
-	if !c.haveObservedExtensionConfigMap {
```

Comment:
where is this gone?

Reply:
moved here https://github.com/openshift/cluster-openshift-apiserver-operator/blob/cfd429184baa311aeda87480f970de72182657a2/pkg/operator/workload/workload_openshiftapiserver_v311_00_sync.go#L105